internal/collector/processor/queued_processor.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package processor
+
+import (
+	"time"
+
+	agenttracepb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/trace/v1"
+	"github.com/jaegertracing/jaeger/pkg/queue"
+	"go.uber.org/zap"
+)
+
+type queuedSpanProcessor struct {
+	queue *queue.BoundedQueue
+	/* TODO: (@pauloja) not doing metrics for now
```

Comment:
delete commented code?

Reply:
gah this scaped my search, notice the wrong alias ;)